### DISCUSSION TITLE
Uncaught ReferenceError: $wire, paginationTotalItemCount, currentlyReorderingStatus, reorderFunction, tableWrapper is not defined

### DISCUSSION THREAD

**Discussion by mandelacruz** [rappasoft/laravel-livewire-tables#1572] [Q&A]
I'm having issues here. The search bar, column selection and number of entries fields are not showing on initial load of the page. But when I click the next page or when I sort the data all fields are displayed. 

Here are the errors I get:
![image](https://github.com/rappasoft/laravel-livewire-tables/assets/37199560/aac94fcf-fc94-482d-a41f-c304bc6d3945)
![image](https://github.com/rappasoft/laravel-livewire-tables/assets/37199560/6f481b35-1025-4421-b743-8dc2f0593ef2)

**Comment by Teckchun** [comment]
I also faced the same issue in version 3.1.x

**Comment by lrljoe** [comment]
@mandelacruz and @Teckchun  
Can you confirm if you're using the "injection" method for both Livewire and the Rappasoft tables please?

In the interim, if you can please try using the "bundler" approach for the Rappasoft Tables assets for now, this works 100%.  The Blade Directives are also consistent!

See here for details on what I'm on about:
https://rappasoft.com/docs/laravel-livewire-tables/v3/start/including-assets

I believe there's been a core change to the Livewire code in a recent Livewire version, that I've missed with the injection approach that is causing it to intermittently get upset.
